src-vue/src/plugins/store.ts

```diff
@@ -184,7 +184,7 @@ function _initializeListeners(state: any) {
  * This retrieves Northstar version tag, and stores it in application
  * state, for it to be displayed in UI.
  */
-async function _get_northstar_version_number(state: any) {
+export async function _get_northstar_version_number(state: any) {
```

Comment:
This method should stay private (not get exported), and should be invoked through the associated store mutation.
Check https://github.com/GeckoEidechse/FlightCore/blob/main/src-vue/src/views/SettingsView.vue#L50 as an example.